packages/web-app-files/tests/unit/helpers/route.spec.js

```diff
@@ -1,48 +1,100 @@
+import {
+  checkRoute,
+  isAnySharedWithRoute,
+  isFavoritesRoute,
+  isPersonalRoute,
+  isPublicFilesRoute,
+  isPublicPage,
+  isSharedWithMeRoute,
+  isSharedWithOthersRoute,
+  isTrashbinRoute
+} from '@files/src/helpers/route.js'
+
 describe('route', () => {
   describe('checkRoute', () => {
-    it.todo('should return "true" if given "routes" array contains given "currentRoute"')
-    it.todo('should return "false" if given "routes" array does not contains given "currentRoute"')
+    it('should return "true" if given "routes" array contains given "currentRoute"', () => {
+      expect(checkRoute(['some-route'], 'some-route')).toBeTruthy()
```

Comment:
if it returns literal `true` then the following should be a good way
```suggestion
      expect(checkRoute(['some-route'], 'some-route')).toBe(true)
```
because I think, with `toBeTruthy()`, test will pass if the function returns anything truthy (like `1`, `"somestring"` and other)